components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -99,6 +99,16 @@ - (void)layoutSubviews {
   [self postLayoutSubviews];
 }
 
+// UITextField's sizeToFit calls this method and then also calls setNeedsLayout.
+// When the system calls this method the size parameter is the view's current size.
+- (CGSize)sizeThatFits:(CGSize)size {
```

Comment:
how did you determine that this was incorrect? was there a tests or snapshot test that was failing? Lets add that so we dont regress on this great improvement.

Reply:
Is it alright if I do a unit test for `-sizeToFit` instead of `-sizeThatFits:`? The former calls the latter, and it just works better with the given/when/then thing we enforce.